Makefile

```diff
@@ -147,6 +147,7 @@ initdb:
 	xz -d -f -k dev/$(DB).sql.xz --stdout | docker-compose run --rm web psql -h db -d warehouse -U postgres -v ON_ERROR_STOP=1 -1 -f -
 	docker-compose run --rm web python -m warehouse db upgrade head
 	$(MAKE) reindex
+	docker-compose run web python -m warehouse sponsors populate-db
```

Comment:
Do we have a plan to run this in production?

As an aside, this is a bunch of work for something we'll only need to do once! Wow!